models/src/main/java/com/vimeo/networking2/Pageable.kt

```diff
@@ -0,0 +1,33 @@
+package com.vimeo.networking2
+
+/**
+ * All lists that can be paged should implement this interface to provide
+ * paging urls, total number of items returns for each page and the data for each page.
+ */
+interface Pageable<DATA_T> {
```

Comment:
for consistency with other type parameters in the library this should be `Data_T` rather than in all caps